configuration/wireless/access-point-routed.md

```diff
@@ -0,0 +1,193 @@
+
+This documentation was tested on a Raspberry Pi 3B running a fresh installation of Raspbian Buster. 
+
+<a name="intro"></a>
+## Before you start
+
+* Ensure you have administrative access to your Raspberry Pi. The network setup will be modified as part of the installation: local access, with screen and keyboard connected to your Raspberry Pi, is recommended.
+* Connect your Raspberry Pi to the Ethernet network and boot the Raspbian OS.
+* Ensure the Raspbian OS on your Raspberry Pi is [up to date](../../raspbian/updating.md) and reboot if packages were installed in the process.
+* Take note of the IP configuration of the Ethernet network the Raspberry Pi is connected to: 
+    * In this document, we assume IP network `10.10.0.0/24` is configured on the Ethernet LAN, and the Raspberry Pi is to manage IP network `192.168.4.0/24` for wireless clients.
+    * Please select another IP network for wireless, e.g. `192.168.10.0/24`, if IP network `192.168.4.0/24` is already in use by your Ethernet LAN.
+* Have a wireless client (laptop, smartphone, ...) ready to test your new access point.
+
+<a name="software-install"></a>
+## Install the access point and network management software
+
+In order to work as an access point, the Raspberry Pi needs to have the `hostapd` access point software package installed:
+
+```
+sudo apt install hostapd
+```
+Enable the wireless access point service and set it to start when your Raspberry Pi boots:
+
+```
+sudo systemctl unmask hostapd
+sudo systemctl enable hostapd
+```
+
+In order to provide network management services (DNS, DHCP) to wireless clients, the Raspberry Pi needs to have the `dnsmasq` software package installed:
+
+```
+sudo apt install dnsmasq
+```
+Finally, install `netfilter-persistent` and its plugin `iptables-persistent`. This utilty helps by saving firewall rules and restoring them when the Raspberry Pi boots:
+
+```
+sudo DEBIAN_FRONTEND=noninteractive apt install -y netfilter-persistent iptables-persistent
+```
+Software installation is complete. We will configure the software packages later on.
+
+<a name="routing"></a>
+## Set up the network router
+
+The Raspberry Pi will run and manage a standalone wireless network. It will also route between the wireless and Ethernet networks, providing internet access to wireless clients. If you prefer, you can choose to skip the routing by skipping the section "Enable routing and IP masquerading" below, and run the wireless network in complete isolation. 
+
+### Define the wireless interface IP configuration
+
+In this document, we assume IP network `10.10.0.0/24` is configured on an Ethernet LAN, and the Raspberry Pi will manage IP network `192.168.4.0/24` for wireless clients.
+**Note:** Please select another IP network for wireless, e.g. `192.168.10.0/24`, if IP network `192.168.4.0/24` is already in use by your Ethernet LAN.
```

Comment:
This sentence seems to be duplicated in the "Before you start" section?

Reply:
Removed.